jekyll/_cci2/roles-and-permissions-overview.adoc

```diff
@@ -291,32 +291,32 @@ The table below shows the permissions associated with each CircleCI organization
 |
 |
 
-^| Create release environment
+^| Create environment integration
+^| icon:check-circle[]
 ^| icon:check-circle[]
-^|
 ^|
 
-^| Delete release environment
+^| Delete environment integration
 ^| icon:check-circle[]
 ^|
 ^|
 
-^| View release environment
+^| View environment integration
 ^| icon:check-circle[]
 ^| icon:check-circle[]
 ^| icon:check-circle[]
 
-^| Create environment token
+^| Create integration token
+^| icon:check-circle[]
 ^| icon:check-circle[]
-^|
 ^|
 
-^| Revoke environment token
```

Comment:
Shouldn't this be "environment integration token"?

Reply:
based on this ticket https://circleci.atlassian.net/browse/REL-3047 should be an integration token.

it was renamed before to that. But I will wait anyways @dlayci  input before merging.

Reply:
For consistency I would agree - we should call it "Environment integration token" everywhere. 

I figured on the environment integration settings page there is enough context there to know it's for the env. integration - but this is lost outside of this context.